onnx_tf/handlers/frontend/bias_add.py

```diff
@@ -24,4 +26,25 @@ def version_6(cls, node, **kwargs):
 
   @classmethod
   def version_7(cls, node, **kwargs):
-    return cls.arithmetic_op(node, **kwargs)
+    data_format = node.attr.get("data_format", "NHWC")
+    channel_first = data_format[1] == "C"
+    axis = 1 if channel_first else -1
+
+    unsqueeze_suffix = get_unique_suffix()
+    if (axis == 1):
+      # In this case, we manually unsqueeze the bias term
+      # to facilitate broadcasting.
+      num_sp_dim = len(data_format) - 2
+      unsqueeze_axes = [i + 1 for i in range(num_sp_dim)]
+      reshape_node = cls.make_node_from_tf_node(
+          node, [node.inputs[1]], [node.inputs[1] + unsqueeze_suffix],
```

Comment:
without underscore as an connector?

Reply:
I think we never used underscore as a connector. It was only used to emphasize the T (transposed) here (https://github.com/onnx/onnx-tensorflow/blob/317d70b763ba91a3eb2e017c2783beb9ff4b3b32/onnx_tf/handlers/frontend/conv_mixin.py). Regardless I think you have a point and this should make the output name more readable.

Reply:
Agree with point of readable.